### DISCUSSION TITLE
short output for switch

### DISCUSSION THREAD

**Discussion by LucaHawtin** [HomeSpan/HomeSpan#688] [Ideas]
Is there a way for the home app to have a switch on the home app send a signal to a relay for half a second for turning something on ie a computer?

**Comment by HomeSpan** [comment]
Yes, this is easily done in HomeSpan.  If your question is in general how do you create HomeSpan sketches, please go through all the tutorials and examples - they teach you almost everything there is about HomeSpan.

However, if you've already created a lot of working HomeSpan sketches, but your question is more specific to implementing a delay that triggers something for 1/2 second after pressing a button in the Home App, this would be done by using the Service `loop()` method.   Tutorial 14 (Emulated Push Button) provides a good example of how to do this.
